#12 Add spring data repository

svgogin opened the pull request:
1. Spring data jdbc repository added
2. Methods to the interface SparkRepositoryDb added (in order to work with Inn instead of id

alinkov commented on src/main/java/ru/svgogin/service/spark/dto/CompanyDto.java:
```diff
@@ -28,30 +28,30 @@ public CompanyDto(String inn,
 
   public String getInn() {
-    return inn;
+    return this.inn;
```
why do you add `this`?

alinkov commented on src/main/java/ru/svgogin/service/spark/entity/Company.java:
```diff
@@ -1,15 +1,28 @@
   private final String statusName;
+  @Column("statusdate")
   private final LocalDate statusDate;
+  @Id
```
please move up id field

alinkov commented on src/main/java/ru/svgogin/service/spark/entity/Company.java:
```diff
@@ -28,30 +41,30 @@ public Company(String inn,
 
   public String getInn() {
-    return inn;
+    return this.inn;
```
also remove `this`

Commit:
#12 Add spring data repository
1. Spring data jdbc repository added
2. Methods to the interface SparkRepositoryDb added (in order to work with Inn instead of id
